fhir-search/src/test/java/com/ibm/fhir/search/parameters/MultiTenantSearchParameterTest.java

```diff
@@ -287,138 +282,26 @@ public void testGetSearchParameter12() throws Exception {
-        assertNull(searchParameter);
-        searchParameter = SearchUtil.getSearchParameter(Patient.class, "favorite-nfl-team");
-        assertNull(searchParameter);
-    }
-
-    @Test
-    void testDynamicSearchParameters2() throws Exception {
-        // Test behavior related to falling back to default tenant.
-        FHIRRequestContext.get().setTenantId("tenant2");
-
-        String mainFile = "target/test-classes/config/tenant2/extension-search-parameters.json";
-        String hiddenFile1 = mainFile + ".hide1";
-
-        // First, remove any copy of the main file that might exist.
-        // Our initial state will be no tenant-specific search parameters present.
-        deleteFile(mainFile);
-
-        // With no tenant-specific search parameters, we should fall back to the default tenant which defines patient
-        // extensions
-        SearchParameter searchParameter = SearchUtil.getSearchParameter(Patient.class, "favorite-mlb-team");
-        assertNull(searchParameter);
-        searchParameter = SearchUtil.getSearchParameter(Patient.class, "favorite-color");
-        assertNotNull(searchParameter);
-
-        List<SearchParameter> result = SearchUtil.getApplicableSearchParameters("Patient");
-        assertNotNull(result);
-        printSearchParameters("testDynamicSearchParameters2/Patient", result);
-        assertEquals(37, result.size());
-
-        // Sleep a bit to allow file mod times to register.
-        Thread.sleep(100);
-
-        // Copy our first hidden file into place.
-        // This should add two tenant-specific search parameters.
-        copyFile(hiddenFile1, mainFile);
-
-        // Verify that we can now see the two tenant-specific search parameters and NOT the default tenant ones
-        searchParameter = SearchUtil.getSearchParameter(Patient.class, "favorite-mlb-team");
-        assertNotNull(searchParameter);
-        searchParameter = SearchUtil.getSearchParameter(Patient.class, "favorite-color");
-        assertNull(searchParameter);
-
-        result = SearchUtil.getApplicableSearchParameters("Patient");
-        assertNotNull(result);
-        printSearchParameters("testDynamicSearchParameters2/Patient", result);
-        assertEquals(33, result.size());
-
-        // Sleep a bit to allow file mod times to register.
-        Thread.sleep(1000);
-
-        // Finally, delete the tenant-specific config file and verify that we fall back to the default ones
-        deleteFile(mainFile);
-        searchParameter = SearchUtil.getSearchParameter(Patient.class, "favorite-mlb-team");
-        assertNull(searchParameter);
-        searchParameter = SearchUtil.getSearchParameter(Patient.class, "favorite-color");
-        assertNotNull(searchParameter);
-
-        result = SearchUtil.getApplicableSearchParameters("Patient");
-        assertNotNull(result);
-        printSearchParameters("testDynamicSearchParameters2/Patient", result);
-        assertEquals(37, result.size());
-    }
```

Comment:
Maybe a negative test to show that it is not dynamic? 
I realize it's highly unlikely, however, it'd keep any incorrect refactors in the future from succeeding.

Reply:
OK, I added one (and only one) of the dynamic tests back in.  It turns out to be a good test for the dynamic / caching behavior thats still in place via ExtensionSearchParametersResourceProvider.